src/altinn-app-frontend/package.json

```diff
@@ -27,41 +27,41 @@
     "@reduxjs/toolkit": "^1.8.1",
     "ajv": "^8.11.0",
     "ajv-formats": "^2.1.1",
-    "axios": "^0.26.1",
+    "axios": "^0.27.2",
     "dot-object": "^2.1.4",
     "immutability-helper": "2.7.0",
     "jsonpointer": "5.0.0",
     "moment": "^2.29.3",
     "react": "^17.0.2",
     "react-content-loader": "^6.2.0",
-    "react-device-detect": "^2.1.2",
+    "react-device-detect": "^2.2.2",
     "react-dom": "^17.0.2",
     "react-dropzone": "^11.7.1",
     "react-number-format": "^4.9.3",
-    "react-redux": "^7.2.6",
-    "react-router-dom": "^5.3.0",
+    "react-redux": "^8.0.1",
+    "react-router-dom": "^5.3.1",
     "redux": "^4.1.2",
     "redux-saga": "^1.1.3",
     "reselect": "^4.1.5",
-    "typescript": "^4.6.3",
+    "typescript": "^4.6.4",
     "uuid": "8.3.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.17.9",
-    "@babel/preset-env": "^7.16.11",
+    "@babel/core": "^7.17.10",
+    "@babel/preset-env": "^7.17.10",
     "@babel/preset-react": "^7.16.7",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.5",
     "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^12.1.4",
-    "@testing-library/user-event": "^13.5.0",
+    "@testing-library/react": "^12.1.5",
```

Comment:
a testing library for hooks was added.

Reply:
Yeah, still present: https://github.com/Altinn/app-frontend-react/blob/a74e3975eb07d337677067464c336fa629b4d47c/src/altinn-app-frontend/package.json#L56